subsys/usb/device_next/class/usbd_cdc_acm.c

```diff
@@ -549,6 +566,10 @@ static void cdc_acm_tx_fifo_handler(struct k_work *work)
 	len = ring_buf_get(data->tx_fifo.rb, buf->data, buf->size);
 	net_buf_add(buf, len);
 
+	if (IS_ENABLED(CONFIG_USBD_CDC_ACM_SHORT_PACKET_TERMINATE_ENABLED)) {
+		data->zlp_needed = (len && ((len & data->ep_mps_mask) == 0));
+	}
+
```

Comment:
https://github.com/zephyrproject-rtos/zephyr/blob/9c0f92db475f0e27673ac366276026eddf1a150c/subsys/usb/device_next/class/usbd_cdc_ecm.c#L522-L524

Reply:
Hi @jfischer-no  Thanks for the pointer. I have adopted this in this PR. 👍